src/EventListener/ShipmentCollectionTimeLockListener.php

```diff
@@ -34,11 +37,12 @@ final class ShipmentCollectionTimeLockListener
     private LockInterface $lock;
     private string $shipmentClass;
 
-    public function __construct(EntityManagerInterface $entityManager, LockInterface $lock, CollectionTimeRepositoryInterface $collectionTimeRepository)
+    public function __construct(EntityManagerInterface $entityManager,  CollectionTimeRepositoryInterface $collectionTimeRepository)
```

Comment:
You should inject the `LockFactory` service instead and use it: https://symfony.com/doc/current/lock.html